menas/src/main/resources/application.properties.template

```diff
@@ -157,3 +157,6 @@ menas.oozie.splineMongoURL=mongodb://localhost:27017
 #javax.net.ssl.trustStorePassword=somePassword
 #javax.net.ssl.keyStore=/path/to/keystore.jks
 #javax.net.ssl.keyStorePassword=somePassword
+
+# Temporary fix for locking of property definition updates #1673
+# menas.feature.property.definitions.update=false
```

Comment:
**Minor"**
```suggestion
# menas.feature.property.definitions.allowUpdate=false
```

Reply:
The life of this prop is gonna be 1 month, but sure.

Reply:
You are right. And it's really minor. So unless you change something else too I would say don't bother.

Reply:
I did it already 😄

Reply:
Ah, already changed meanwhile. 🙇 (Sorry for the unneeded nitpicking)